<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>311</width>
      <height>332</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelLoggedUser">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Zalogowano jako: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddTip">
       <property name="text">
        <string>Dodaj napiwek</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditLastTip">
       <property name="text">
        <string>Edytuj ostatni napiwek</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCheckBalance">
       <property name="text">
        <string>Sprawdz saldo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnManagerTools">
       <property name="text">
        <string>Sprawdz saldo innych kelnerów</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdminPanel">
       <property name="text">
        <string>Zarządzanie pracownikami</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPerformanceMetrics">
       <property name="text">
        <string>Sprawdź metryki kelenerów</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBackup">
       <property name="text">
        <string>Utwórz kopię zapasową</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGenerateReports">
       <property name="text">
        <string>Generuj raporty miesięczne</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLogout">
       <property name="text">
        <string>Wyloguj</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
